src/models/popUp.js

```diff
@@ -194,10 +198,36 @@ const popUpModel = {
             // TODO: try making a 1D growth comparator
 
             // TODO: position = absolute or fixed
+            function absolutePosition(fnode) {
+                return ((fnode.element.style.position == "absolute") ? ONEISH : ZEROISH) ** coeffAbsolutePosition;
+            }
             // TODO: align items: centered
 
+
+
             // TODO: try z-index again
+            
+            function getMaxZIndex() {
+                console.log('should only go into this once???');
+                 return Array.from(document.querySelectorAll('body *'))
+                       .map(a => parseFloat(window.getComputedStyle(a).zIndex))
+                       .filter(a => !isNaN(a))
+                       .sort()
```

Comment:
This will sort lexically. [1, 2, 10] will sort as [1, 10, 2]. Have I mentioned that JS is a horrible language? Try `.sort((a, b) => a - b)`.